app/src/main/java/org/simple/clinic/summary/assignedfacility/AssignedFacilityEffectHandler.kt

```diff
@@ -20,8 +20,20 @@ class AssignedFacilityEffectHandler(
   fun build(): ObservableTransformer<AssignedFacilityEffect, AssignedFacilityEvent> = RxMobius
       .subtypeEffectHandler<AssignedFacilityEffect, AssignedFacilityEvent>()
       .addTransformer(LoadAssignedFacility::class.java, loadAssignedFacility())
+      .addTransformer(ChangeAssignedFacility::class.java, changeAssignedFacility())
       .build()
 
+  private fun changeAssignedFacility(): ObservableTransformer<ChangeAssignedFacility, AssignedFacilityEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulersProvider.io())
+          .map { (patientUuid, assignedFacilityId) ->
```

Comment:
Is `map` the right operator to use here since we are not using the result of `PatientRepository#updateAssignedFacility` anywhere? We could use a `.doOnNext` instead.